Fix crash when using Emacs key bindings to move the cursor

I seem to be able to trigger the following crash through a combination of ^A, arrow keys, and backspace.

```
panic: runtime error: slice bounds out of range
goroutine 6 [running]:
panic(0x812c40, 0xa9be90)
        /data/pedro/go/src/runtime/panic.go:531 +0x1cf
github.com/companyzero/ttk.(*Edit).KeyHandler(0xc42012e2d0, 0x7f0000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xc)
        /data/pedro/git/go/src/github.com/companyzero/ttk/edit.go:166 +0x576
github.com/companyzero/ttk.(*Window).keyHandler(0xc4200653e0, 0x7f0000, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0xc4200dc4a0, ...)
        /data/pedro/git/go/src/github.com/companyzero/ttk/window.go:237 +0xb8
github.com/companyzero/ttk.initKeyHandler.func1()
        /data/pedro/git/go/src/github.com/companyzero/ttk/ttk.go:332 +0x11e
github.com/companyzero/ttk.init.4.func1(0xc420011340, 0xc420058fc0, 0xc420011298, 0xc4200c67a0)
        /data/pedro/git/go/src/github.com/companyzero/ttk/ttk.go:288 +0x100
created by github.com/companyzero/ttk.init.4
        /data/pedro/git/go/src/github.com/companyzero/ttk/ttk.go:292 +0x234
zsh: exit 2     zkc
```

--- a/edit.go
+++ b/edit.go
@@ -107,7 +107,7 @@ func (e *Edit) KeyHandler(ev termbox.Event) bool {
 	case termbox.KeyCtrlE, termbox.KeyEnd:
 		if len(e.display) < e.trueW-1 {
 			// no need to call display
-			e.cx = e.trueX + len(e.display)
+			e.cx = e.trueX + len(e.display) - e.at
 			setCursor(e.cx, e.cy)
 			return true
 		}

